static/css/post.less

```diff
@@ -413,6 +413,36 @@
 .post_body {
     .u-clearfix();
     color: @darkgray;
+
+    // Spacing has been eyeballed to be 40px.
+    p ~ h1, p ~ .h1 {
+        margin-top: unit(22px / @base-font-size-px, em);
+    }
+
+    // Spacing has been eyeballed to be 40px.
+    p ~ h2, p ~ .h2 {
+        margin-top: unit(31px / @base-font-size-px, em)
+    }
+
+    // Spacing has been eyeballed to be 40px.
+    p ~ h3, p ~ .h3 {
+        margin-top: unit(35px / @base-font-size-px, em)
+    }
+
+    // Spacing has been eyeballed to be 15px.
+    p ~ h4, p ~ .h4 {
+        margin-bottom: unit(7px / @base-font-size-px, em);
```

Comment:
Yeah, I'm wondering if just the offsets should be pulled out so it's clear what the intended values are? For example, for the 40px gaps:

```
margin-top: unit( (40px - 9px) / @base-font-size-px, em);

...

margin-top: unit( (40px - 5px) / @base-font-size-px, em);
```

This can be a larger discussion for elsewhere though, because this adjustment happens in other places also. I remember adding a PR that has an adjustment like this.